### ISSUE TITLE
Enable accesibility for keyboard shortcuts not working

### ISSUE THREAD

**Issue by Docxavier** [the0neyouseek/MonitorControl#123]
I cannot enable MonitorControl in System Preferences > Security and Privacy > Accessibility for the keyboard shortcuts to work on Catalina Beta

**Comment by JoniVR** [comment]
Can you give us more details? What happens? Does MonitorControl show up there or not?

**Comment by Docxavier** [comment]
Hi Joni,

Wen the app starts, it’s asking about this



When I try to click on the box to enable the app in Preferences>Security & …., the click on the box doesn’t work, so it’s not possible to enable the app for the keyboard shortcuts to work.


MonitorControl runs but without shortcuts. 

Am I doing something wrong?

**Comment by JoniVR** [comment]
Can you try removing MonitorControl from the list inside Security & Privacy? And after that restart MonitorControl and try to enable it again.

A gif of what's happening would be helpful too :)

**Comment by Docxavier** [comment]
> Can you try removing MonitorControl from the list inside Security & Privacy? And after that restart MonitorControl and try to enable it again.
> 
> A gif of what's happening would be helpful too :)

Ops, this works ... Now I can set the app correctly

I'm not using apple Kb, could I use customized keys?

Many thanks¡¡¡

**Comment by JoniVR** [comment]
> I'm not using apple Kb, could I use customized keys?

Not as of yet, but it has been requested already, see #118  :)

**Comment by yanghoonkim** [comment]
It also works for me!
